### PR TITLE
Stabilization/2310 -> Development (10/03/2023 Edition)

### PR DESCRIPTION
**Pull request by AMZN-Olex**
## What does this PR do?

Stabilization/2310 @ 9f9829b26a47c2f255959b8e7edac24ac081c651 -> Development

## How was this PR tested?

Testing locally with o3de-multiplayersample and running AR. In progress...

### CODE REVIEW

File: .automatedtesting.json [hunk 1]
```diff
@@ -12,10 +12,5 @@
     "ENABLE_GEMS": [
     ],
     "REPOS": [
-        {
-            "NAME": "o3de-extras",
-            "URL": "https://github.com/o3de/o3de-extras.git",
-            "BRANCH": "development"
-        }
```
Comment (lumberyard-employee-dm) [inline .automatedtesting.json:19]: Calling @amzn-changml 
This change should not be merged over. 
It is part of the .gitignore in the stabilization/2310 repo and it's purpose is to stop o3de-extras from being built in the stabilization branch

**Review by lumberyard-employee-dm — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -14,6 +14,7 @@ CMakeUserPresets.json
 [Uu]ser/
 FrameCapture/**
 .DS_Store
+.automatedtesting.json
```
Comment (lumberyard-employee-dm) [inline .gitignore:17]: This change is part of the change above to disable using the o3de-extras repo when building the stabilization branch and should not be merged over.

Comment (AMZN-Olex) [reply .gitignore]: Done, anything else?

File: .automatedtesting.json (hunk 1 above)
Comment (AMZN-Olex) [reply .automatedtesting.json]: Done, anything else?

Comment (lumberyard-employee-dm) [reply .automatedtesting.json]: No, this should be good.

**Review by lumberyard-employee-dm — APPROVED** [review]

**Review by cgalvan — APPROVED** [review]

File: .automatedtesting.json (hunk 1 above)
Comment (amzn-changml) [reply .automatedtesting.json]: Yup, thanks!

**Review by amzn-changml — APPROVED** [review]

**Comment by amzn-changml** [comment]
Looks like a few commits (including mine) lost their signoff somehow. I can make a remediation commit or you can try it using the instructions here: https://github.com/o3de/o3de/pull/16844/checks?check_run_id=17358801010

**Review by moudgils — APPROVED** [review]

**Pull request merged**